sfge/src/main/java/com/salesforce/graph/build/StaticBlockUtil.java

```diff
@@ -64,7 +64,8 @@
  *          SyntheticStaticBlock_2();
  *     }
  * }
- * </code>
+ * </code> TODO: Some of the helper methods in this class can be pulled into {@link
```

Comment:
New tech debt. There's some redundancy in how `StaticBlockUtil` and the new `VertexSynthesisUtil` create vertices. In the fullness of time, we should probably aim to combine them into the same class.
Alternatively, I can do some of that as part of this PR if that's preferable.